notebooks/book1/03/sensor_fusion_2d.ipynb

```diff
@@ -0,0 +1,503 @@
+{
```

Comment:
<p>Isn't the font size too small? was this creating a problem? Ideally we should start with default 9 and iteratively decrease it if it is creating problems.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Yea I agree it is small but given the way the graph was organized, I just managed to tuck it in the upper left corner using the font_size=6. I will try to see if I can maybe make it slightly larger